x/pairing/keeper/staking.go

```diff
@@ -92,12 +95,31 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, validator, creator, chainID strin
 		}
 		details = append(details, utils.Attribute{Key: "moniker", Value: moniker})
 
+		if !existingEntry.DelegateTotal.IsZero() {
+			diff := ctx.BlockTime().UTC().Unix() - int64(existingEntry.LastChange)
+			_ = diff
+			if ctx.BlockTime().UTC().Unix()-int64(existingEntry.LastChange) < int64((time.Hour * 24).Seconds()) {
```

Comment:
why are you not using `diff`?
also, make the 24H a constant

Reply:
https://github.com/lavanet/lava/pull/1263/commits/5cda7df3147294fb3b1aaa960ff157f5d8e2b00b